### ISSUE TITLE
Windows support

### ISSUE THREAD

**Issue by rhacker** [roynalnaruto/svm-rs#8]
I'm installing forge and get this lib as deps, I got this error message: 
```
   Compiling svm-rs v0.2.0
error[E0433]: failed to resolve: could not find `unix` in `os`
 --> C:\Users\<user>\.cargo\registry\src\github.com-1ecc6299db9ec823\svm-rs-0.2.0\src/lib.rs:9:9
  |
9 |     os::unix::fs::PermissionsExt,
  |         ^^^^ could not find `unix` in `os`

error[E0599]: no function or associated item named `from_mode` found for struct `Permissions` in the current scope
   --> C:\Users\<user>\.cargo\registry\src\github.com-1ecc6299db9ec823\svm-rs-0.2.0\src/lib.rs:136:40
    |
136 |         f.set_permissions(Permissions::from_mode(0o777))?;
    |                                        ^^^^^^^^^ function or associated item not found in `Permissions`

Some errors have detailed explanations: E0433, E0599.
For more information about an error, try `rustc --explain E0433`.
error: could not compile `svm-rs` due to 2 previous errors
warning: build failed, waiting for other jobs to finish...
error: failed to compile `foundry-cli v0.1.0 (https://github.com/gakonst/foundry#36356823)`, intermediate artifacts can be found at `C:\Users\<user>\AppData\Local\Temp\cargo-install4uzvr9`
```

Can we make this work with Windows? Thanks

**Comment by gakonst** [comment]
It's WIP in #9

**Comment by roynalnaruto** [comment]
Hi @rhacker , @mattsse 's PR was merged and I've published 0.2.1 of svm-rs. Could you give it a try again?

**Comment by rhacker** [comment]
ok, I'm trying it now.
